base/abstractarray.jl

```diff
@@ -1218,6 +1218,7 @@ end
 copymutable(itr) = collect(itr)
 
 zero(x::AbstractArray{T}) where {T<:Number} = fill!(similar(x, typeof(zero(T))), zero(T))
+zero(x::AbstractArray{S}) where {T<:Number, S<:Union{Missing, T}} = fill!(similar(x, typeof(zero(S))), zero(S))
```

Comment:
Arguably this should be:
```suggestion
zero(x::AbstractArray{<:Union{Missing, T}}) where {T<:Number} = fill!(similar(x), zero(T))
```
which would mean that you continue to get back a type which can be mutated to hold `missing`s.
However that is not the behavour of 1.10.
So I didn't do that.